Makefile

```diff
@@ -332,7 +332,7 @@ markdownlint-cli:
 	which markdownlint || npm install -g --omit=dev markdownlint-cli
 
 .PHONY: deploy-autoscaler deploy-register-cf deploy-autoscaler-bosh deploy-cleanup
-deploy-autoscaler: go-mod-vendor uaac db scheduler deploy-autoscaler-bosh deploy-register-cf
```

Comment:
All these dependencies to the `download-java-bosh-package` should not be necessary, same as they are not necessary for the Go package

Reply:
Great... I did not know that part.. Thanks for sharing

Reply:
done